src/Form/Type/UserType.php

```diff
@@ -49,7 +50,11 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             ->add('organization', OrganizationEntityType::class)
             ->add('firstName', TextType::class)
             ->add('lastName', TextType::class)
-            ->add('phoneNumber', TextType::class)
+            ->add('phoneNumber', TextType::class, [
+                'constraints' => [
+                    new Regex(['pattern' => '#^0[1-9][0-9]{8}$#', 'message' => 'Le format est invalide, exemple : 0601020304.']),
```

Comment:
Don't you think we should put this check on the entity property, with an Assert ?